docs/api/moving-the-robot.md

```diff
@@ -0,0 +1,64 @@
+# Moving the Robot
+
+This page describes the various APIs available to move the Create® 3 robot via ROS 2[^1].
+
+!!! important
+    Supervise your robot while executing movements, to avoid risk of damaging it or its surroundings.
+
+!!! important
+    It's recommended to not try to command robot motions if the robot is on its docking station.
+    Either lift it and place it somewhere else or use the built-in [undocking behavior](../docking/#undocking).
+
+
+## Velocity Control
+
+The velocity control API is the simplest way to move the Create® 3 robot.
+It's recommended for beginners and to simply try out the robot.
+Being the lowest-level motion API exposed by the robot, this is also recommended to the more advanced users who want to implement their own autonomous navigation applications.
+
+The robot is controlled by publishing [geometry_msgs/msg/Twist](https://github.com/ros2/common_interfaces/blob/rolling/geometry_msgs/msg/Twist.msg) messages to the `/cmd_vel` topic.
+For example, to move the robot straight forward using the ROS 2 command line:
+
+```bash
+ros2 topic pub -r 20 /cmd_vel geometry_msgs/msg/Twist "{linear: {x: 0.2, y: 0.0, z: 0.0}, angular: {x: 0.0, y: 0.0, z: 0.0}}"
+```
+
+The robot will keep executing the last received message until a new message arrives, which will be executed from from now, or a pre-fixed timeout expires without new messages being received.
```

Comment:
The syntax on this sentence is a little confusing and I'm not sure I fully understand. Does this encapsulate what you are trying to say?

> The robot will execute the last received message on the topic until a fixed timeout period has passed or a new message is received, at which point the timer is restarted and this process begins again.